10.Mender-server/docs.md

```diff
@@ -7,6 +7,5927 @@ shortcode-core:
 github: false
 ---
 
+## Mender 3.2.0
+
+_Released 01.24.2022_
+
+### Statistics
+
+A total of 77976 lines added, 32213 removed (delta 45763)
```

Comment:
I find it hard to believe these numbers. Is Mender 3.2 (server) really like 4x [Mender 3.1 (server)](https://docs.mender.io/release-information/release-notes-changelog/mender-server#statistics)? And Kristian being in the second position for most lines? Might it be that we are counting all components here not only the server?

Reply:
So, looked into it. The reason is the new enterprise service (`deviceauth`).

This does aggregate the whole history.

tbh, I'm not completely sure what's actually supposed to be correct here.

I see why the whole history is included...

Reply:
On new services we usually remove the changelog and just write "First release of <service>". That will get the rid of the enormous (and wrong) changelog.

And with regards to the statistics, I don't think we can fix it that easily. Maybe we have left it like that in the past when we have introduced an `-enterprise` fork. The numbers are just so skewed...